helpers/helper.js

```diff
@@ -0,0 +1,86 @@
+const fs = require("fs");
+
+/**
+ * This function generates a new id
+ *
+ * @param  {Array} array The movies array
+ * @returns {number} The generated id
+ */
+const getNewId = (array) => {
+  if (array.length > 0) {
+    return array[array.length - 1].id + 1;
+  } else {
+    return 1;
+  }
+};
+
+/**
+ * This function generates a new date
+ *
+ * @returns {string} The generated date
+ */
+const newDate = () => new Date().toString();
+
+/**
+ * This function checks if a movie is in the object based on its id
+ *
+ * @param  {Array} array The movies array
+ * @param  {number} id The movie id
+ * @returns {object} The movie
+ */
+function mustBeInArray(array, id) {
```

Comment:
The name implies that the method checks simething and by convention such methods must return a boolean value.